week3/homework/src/index.js

```diff
@@ -1,3 +1,99 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+const express = require('express');
+const bodyParser = require('body-parser');
+const functions = require('./functions.js');
+const app = express();
+app.use(bodyParser.json());
+
+app.get('/todos/:id', (request, response) => {
+  const index = request.params.id;
+  fs.readFile('./src/to-do.json', 'utf8', (error, data) => {
```

Comment:
You are using `fs.readFile()` in multiple routes of your application: you should move this functionality to a function (like `getData()`) which you can call from the different routes.

Reply:
Tried but then it doesnt work at all